app/src/main/res/values-it/help.xml

```diff
@@ -51,32 +51,32 @@
-        will need it to access your wallet again.<br/>
+        <h2>Password di ripristino dei file del portafoglio</h2>
+        Assicurati di aver salvato questa password. Se resetti il tuo dispositivo o disinstalli l'app
+        ne avrai bisogno per accedere di nuovo al tuo portafoglio.<br/>
         <h3>CrAzYpass</h3>
-        If the password displayed here is 52 alphanumeric characters in groups of 4 - congratulations!
-        Your wallet files are secured with a 256-bit key generated by your device&apos;s security
-        features based on the passphrase you chose (on creation or by changing it). This makes it
-        extremenly difficult to hack!<br/>
-        This feature is mandatory for all newly created wallets.
-        <h3>Legacy Password</h3>
-        If you see your passphrase here, your wallet files are not as secure as when using
-        a CrAzYpass. To fix this, simply select \"Change Passphrase\" from the menu. After entering
-        a new passphrase (maybe even the same one as before) the app will generate a CrAzYpass for
-        you and secure your wallet files with it. Write it down!
-        <h3>CrAzYpass wallets</h3>
-        If you ever need to reinstall Monerujo (for example after resetting your phone or switching
-        to a new one) or you want to use your wallet files on a different device or PC, you have to
-        use this Recovery Password in order to access your wallet again.<br/>
-        By selecting \"Change Passphrase\" from the menu, you can choose another passphrase. Beware
-        that this will generate a new Recovery Password. Write it down!
+        Se la password visualizzata qui contiene 52 caratteri alfanumerici in gruppi di 4 - Congratulazioni!
+        I file del tuo portafoglio sono criptati con una chiave a 256-bit generata dal tuo dispositivo
+        e basata sulla passphrase che hai scelto (alla creazione o quando l'hai modificata). Questo la rende estremamente
+        difficile da hackerare!<br/>
+        Questa feature è obbligatoria per tutti i nuovi portafogli.
+        <h3>Password precedente</h3>
+        Se vedi la tua passphrase qui, i file del tuo portaglio non sono così sicuri come 
+        quelli criptati da CrAzYpass. Per adeguare la tua password semplicemente seleziona \"Cambia Passphrase\" dal menu.
+        Dopo aver inserito una nuova passphrase (perfino la stessa usata in precedenza) l'applicazione genererà
+        una CrAzYpass per te e la userà per mettere al sicuro i file del tuo wallet. Segnatela!
+        <h3>Portafogli CrAzYpass</h3>
+        Se dovessi avere bisogno di reinstallare Monerujo (per esempio dopo aver resettato il tuo telefono
+        o dopo averlo sostituito) o vuoi usare i file del tuo portafoglio su un diverso dispositivo o PC, devi
+        usare questa password di recupero per poter accedere nuovamente al tuo portafoglio.<br/>
+        Selezionando \"Cambia Passphrase\" dal menu, potrai scegliere un'altra passphrase. Questa azione
```

Comment:
@m2049r kindly asked to replace " ' " with the HTML tag. Dunno if it is applicable here too.

Reply:
yes, please: ```\'```
Android is a bit weird that way.

Reply:
Sorry, i completely forgot about that